PluginCore/PluginCore/Utilities/PathWalker.cs

```diff
@@ -98,8 +98,13 @@ private void ExploreFolderWithMasks(String path, String[] masks)
             foreach (String mask in masks)
             {
                 String[] files = Directory.GetFiles(path, mask);
+                String control = mask.Length == 5 && mask[0] == '*' && mask[1] == '.' ? mask.Substring(1) : null;
```

Comment:
I'd say this won't work if mask is something like *.hx*, *.hx?, *f*.hx, etc. PluginCore has Helpers.FileHelper.FileMatchesSearchFilter, although I'd say it would be more costly for something like this where you only want to check the extension (the extension part could be extracted tho).